sickbeard/search/queue.py

```diff
@@ -312,8 +321,8 @@ def run(self):
                        format(('forced', 'manual')[bool(self.manual_search)],
                               ('', 'season pack ')[bool(self.manual_search_type == 'season')], self.segment[0].pretty_name()))
 
-            search_result = search.searchProviders(self.show, self.segment, True, self.downCurQuality,
-                                                   self.manual_search, self.manual_search_type)
+            search_result = searchProviders(self.show, self.segment, True, self.downCurQuality,
+                                                 self.manual_search, self.manual_search_type)
```

Comment:
Still doing some cleanup on these, but wanted to post it so that anyone wanting to review could see where i was at with them.